src/foam/dao/OnDeleteRemoveChildrenDAO.js

```diff
@@ -0,0 +1,58 @@
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'OnDeleteRemoveChildrenDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  javaImports: [
+    'foam.mlang.MLang'
+  ],
+
+  properties: [
```

Comment:
These properties are all stored in the Relationship. Shouldn't/couldn't we just use the Relationship instead?